pennylane_lightning/lightning_gpu/lightning_gpu.py

```diff
@@ -504,10 +503,13 @@ def _resolve_mcm_method(mcm_config: MCMConfig, tape: None | qml.tape.QuantumScri
         final_mcm_method = "one-shot" if getattr(tape, "shots", None) else "deferred"
     elif mcm_config.mcm_method == "device":
         final_mcm_method = "tree-traversal"
+
+    if mcm_config.postselect_mode == "fill-shots" and final_mcm_method != "deferred":
+        raise DeviceError("Using postselect_mode='fill-shots' is not supported.")
```

Comment:
```suggestion
        raise DeviceError("Using postselect_mode='fill-shots' is only supported with mcm_method='deferred'.")
```

Is this more accurate?

Reply:
Although `"deferred"` not supported with postselection is a missing feature. There's nothing fundamentally wrong with it. That's why the `if` statement above is still what it is. I thought I didn't want to confuse the user with "hey you need to use `"deferred"`", and when they do, it still doesn't work. Maybe in the future when `"deferred"` + postselection support is actually added to lightning, we can update this error message to be more specific.